src/providers/google-vertex-ai/chatComplete.ts

```diff
@@ -253,12 +266,20 @@ export const VertexGoogleChatCompleteConfig: ProviderConfig = {
     default: '',
     transform: (params: Params) => {
       const functionDeclarations: any = [];
+      const tools: any = [];
       params.tools?.forEach((tool) => {
         if (tool.type === 'function') {
-          functionDeclarations.push(tool.function);
+          if (tool.function.name === 'googleSearchRetrieval') {
+            tools.push(buildGoogleSearchRetrievalTool(tool));
+          } else {
+            functionDeclarations.push(tool.function);
+          }
```

Comment:
Is it possible add `strictOpenAiCompliance` compliance here as well?

Reply:
Hey @b4s36t4 strict open ai compliance flag is used for response transforms, it is currently not used in request config object